OpenRA.Mods.Cnc/Traits/PortableChrono.cs

```diff
@@ -55,6 +55,18 @@ class PortableChronoInfo : TraitInfo, Requires<IMoveInfo>
 		[VoiceReference]
 		public readonly string Voice = "Action";
 
+		[Desc("Range circle outline contrast color")]
+		public readonly Color CircleColor = Color.FromArgb(128, Color.LawnGreen);
+
+		[Desc("Range circle line width")]
+		public readonly float CircleWidth = 1;
+
+		[Desc("Range circle outline contrast color")]
```

Comment:
Can you please update `s/contrast/border` in the descriptions too?

Reply:
Yes.